### ISSUE TITLE
Lost fully qualified type (version 1.3.6)

### ISSUE THREAD

**Issue by lsoft** [zompinc/sync-method-generator#51]
Hello.

Here is one additional code snippet with lost fully qualified name.

```C#
using OtherNamespace;

namespace SomeNamespace
{
    public partial class Subject1
    {
        [Zomp.SyncMethodGenerator.CreateSyncVersion]
        public async Task<(XP, bool)> FooAsync()
        {
            throw new NotImplementedException();
        }
    }
}

namespace OtherNamespace
{
    public class XP {}
}
```

The problem:

![image](https://github.com/zompinc/sync-method-generator/assets/5988558/2408b5ca-4d09-4a4e-b498-d82a19993ea5)

**Comment by virzak** [comment]
I'm working on a rewriter which is a standalone and only takes care of fully qualifying everything.

I'll have this fixed this week.

**Comment by virzak** [comment]
Fixed in version 1.3.13
